tools/castle-editor/code/formproject.pas

```diff
@@ -827,16 +930,71 @@ procedure TProjectForm.FormCreate(Sender: TObject);
   BuildPlatformsMenu;
   BuildPackageFormatsMenu;
   ApplicationProperties.OnWarning.Add(@WarningNotification);
+  if IsDockUIEnabled then
+  begin
+    // Create dockable forms
+    //  DockMaster.DockSitesCanBeMinimized := True;
+    DockMaster.MakeDockSite(Self, [akBottom], admrpNone);
+    DesignForm := TDesignForm.Create(nil);
+    DesignHierarchyForm := TDesignHierarchyForm.Create(nil);
+    DesignPropertiesForm := TDesignPropertiesForm.Create(nil);
+    DesignExplorerForm := TDesignExplorerForm.Create(nil);     
+    DesignFilesForm := TDesignFilesForm.Create(nil);
+    DesignOutputForm := TDesignOutputForm.Create(nil);
+    DesignWarningsForm := TDesignWarningsForm.Create(nil);
+    DockMaster.MakeDockable(DesignForm, True, True);
```

Comment:
I remember I went with B before, but switch back to current choice because AnchorDocking somehow not able to show resourceless forms, even though they did show perfectly if I didn't add them to DockMaster.

Reply:
Done. Maybe that issue only occurs when I use Lazarus trunk.